src-grpc-web-client/elevated-sidecar_pb.ts

```
@@ -365,6 +378,80 @@ class SetErrorReportingEnabledRequest$Type extends MessageType<SetErrorReporting
+      end = reader.pos + length;
+    while (reader.pos < end) {
+      let [fieldNo, wireType] = reader.tag();
+      switch (fieldNo) {
+        case /* bool installation_retained */ 1:
+          message.installationRetained = reader.bool();
+          break;
+        case /* uint32 cleanup_process_id */ 2:
+          message.cleanupProcessId = reader.uint32();
+          break;
+        default:
+          let u = options.readUnknownField;
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)